libs/ui-toolkit/src/components/input-error/input-error.tsx

```diff
@@ -39,6 +39,7 @@ export const InputError = ({
       data-testid="input-error-text"
       className={effectiveClassName}
       {...props}
+      role="alert"
```

Comment:
follow up (non-blocking): we should we add eslint react a11y plugin.

Reply:
https://github.com/vegaprotocol/frontend-monorepo/issues/210